### ISSUE TITLE
PHP-Error (GLPI 10.0.5 - 1.18.2)

### ISSUE THREAD

**Issue by tsmr** [pluginsGLPI/fields#597]
```
[2022-11-21 16:51:32] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "status" in /var/www/html/glpi/marketplace/fields/inc/container.class.php at line 1239
  Backtrace :
  marketplace/fields/inc/container.class.php:1055    PluginFieldsContainer::validateValues()
  ...place/metademands/inc/metademand.class.php:1945 PluginFieldsContainer->updateFieldsValues()
  marketplace/metademands/inc/wizard.class.php:3381  PluginMetademandsMetademand->addObjects()
  marketplace/metademands/inc/wizard.class.php:475   PluginMetademandsWizard::createMetademands()
  marketplace/metademands/inc/wizard.class.php:440   PluginMetademandsWizard::showWizardSteps()
  marketplace/metademands/front/wizard.form.php:688  PluginMetademandsWizard->showWizard()
```

```
$status_overrides = $data[$status_field_name] !== null
            ? PluginFieldsStatusOverride::getOverridesForItemtypeAndStatus($container->getID(), $itemtype, $data[$status_field_name])
            : [];
```
must be replace by

```
$status_overrides = (isset($data[$status_field_name]) && $data[$status_field_name] !== null)
            ? PluginFieldsStatusOverride::getOverridesForItemtypeAndStatus($container->getID(), $itemtype, $data[$status_field_name])
            : [];
```

**Comment by cedric-anne** [comment]
@tsmr 

It is probably fixed in v1.19.0, could you confirm ?

**Comment by stonebuzz** [comment]
There has been no activity on this issue for some time and therefore it is considered stale

If this issue is related to a bug, please try to reproduce on latest release (GLPI and plugin). If the problem persist, feel free to open a new issue.

If it is related to a new feature, please open a topic to discuss with community about this enhancement on [suggestion website](https://glpi.userecho.com/).

You may also consider taking a [subscription](https://glpi-project.org/subscriptions/) to get professionnal support or [contact GLPI editor team](https://portal.glpi-network.com/contact-us) directly.
